src/vue-components/infl-footnote.vue

```diff
@@ -63,7 +66,39 @@
 
         target.setAttribute('data-x', x)
         target.setAttribute('data-y', y)
+      },
+
+      isOutOfRightXBound(childBR, parentBR) {
+        return (childBR.x + childBR.width) > (parentBR.x + parentBR.width)
+      },
+
+      isOutOfLeftXBound(childBR, parentBR) {
+        return childBR.x < 0
+      },
+
+      checkBounds () {
+        let inflpopup = this.$el.querySelector('.alpheios-inflections__footnote-popup')
+        let inflpanel = this.$el.closest('#alpheios-panel__inflections-panel')      
+
+        let popupBR = inflpopup.getBoundingClientRect()
```

Comment:
same question as I had about Kirill's caching comment - is it possible to have multiple footnotes open at once? If so, does this work since it only works on the first?

Reply:
Yes, @balmas - there is no problems with it.

Reply:
ok, thanks! I don't have any further comments to add to Kirill's then.